geo/Cargo.toml

```diff
@@ -31,7 +31,7 @@ proj = { version = "0.27.0", optional = true }
 robust = "1.1.0"
 rstar = "0.12.0"
 serde = { version = "1.0", optional = true, features = ["derive"] }
-i_overlay = { version = "1.7.2, < 1.8.0", default-features = false }
+i_overlay = { version = "1.9.0, < 1.10.0", default-features = false }
```

Comment:
In light of https://github.com/iShape-Rust/iOverlay/issues/12#issuecomment-2504252348, should this be

```diff
i_overlay = { version = "=1.9.0", default-features = false }
```
?

Reply:
https://github.com/iShape-Rust/iOverlay?tab=readme-ov-file#versioning-policy says:

> PATCH updates (e.g., 1.8.1 → 1.8.2): Guaranteed to be backward-compatible, containing only bug fixes or small improvements.

So I think we're good allowing PATCH updates.